test/config/.env.override

```diff
@@ -0,0 +1,2 @@
+MONGODB_URI=mongodb://mongodb:27017/scicat-test-db
+ES_HOST=http://elastic:9200
```

Comment:
I think the `.env.override` is expected to be mounted instead of having it in the core

https://github.com/SciCatProject/scicat-backend-next/pull/1889